packages/client/helpers/build.ts

```diff
@@ -6,15 +6,13 @@ import { build } from '../../../helpers/compile/build'
 import { fillPlugin } from '../../../helpers/compile/plugins/fill-plugin/fillPlugin'
 
 // we define the config for runtime
-const runtimeBuildConfig: BuildOptions = {
+const nodeRuntimeBuildConfig: BuildOptions = {
   name: 'runtime',
   entryPoints: ['src/runtime/index.ts'],
   outfile: 'runtime/index',
   bundle: true,
   define: {
-    'globalThis.NOT_PRISMA_DATA_PROXY': 'true',
-    // that fixes an issue with lz-string umd builds
-    'define.amd': 'false',
```

Comment:
Looks like this issue does not exist anymore, it was probably because of the previous build hacks.